website/docusaurus.config.js

```diff
@@ -6,8 +6,8 @@ const darkCodeTheme = require('prism-react-renderer/themes/dracula');
 
 /** @type {import('@docusaurus/types').Config} */
 const config = {
-  title: 'OneTable',
-  favicon: 'images/onetable_favicon.png',
+  title: 'XTable',
```

Comment:
I guess references here should also be "Apache XTable™"

Reply:
thanks for catching, I updated